packages/react-composites/src/composites/MeetingComposite/styles/SidePane.styles.ts

```diff
@@ -9,7 +9,7 @@ const palette = theme.palette;
 export const sidePaneContainerStyles: IStackItemStyles = {
   root: {
     height: '100%',
-    minWidth: '21.5rem'
+    width: '21.5rem'
```

Comment:
note: this is not good with responsive design in mind, but that will be a future task